docs/usage/configuration-options.md

```diff
@@ -3466,6 +3450,38 @@ This is why we configured an upper limit for how long we wait until creating a P
 !!! note
     If the option `minimumReleaseAge` is non-zero then Renovate disables the `prNotPendingHours` functionality.
 
+## prOptions
+
+Use `prOptions` to set pr-specific configuration options provided by Renovate.
+
+Example:
+
+```json
+{
+  "prOptions": {
+    "gitLabIgnoreApprovals": true
+  }
+}
+```
+
+You can use these options:
+
+### azureWorkItemId
+
+When creating a PR in Azure DevOps, some branches can be protected with branch policies to [check for linked work items](https://docs.microsoft.com/en-us/azure/devops/repos/git/branch-policies?view=azure-devops#check-for-linked-work-items).
+Creating a work item in Azure DevOps is beyond the scope of Renovate, but Renovate can link an already existing work item when creating PRs.
```

Comment:
```suggestion
Renovate will not create a work item in Azure DevOps, this is beyond the scope of Renovate.
But Renovate can _link_ to an existing work item when Renovate creates a PR.
```

Split up sentence into two parts.

Can you configure a `azureWorkItemId` and then forget about it, as it just works? Or do you need to update the `azureWorkItemId` later?